config/config_wcoss2.sh

```diff
@@ -35,6 +35,7 @@ export WGET="wget -nv"
 # define cc, FC and CC for CC, FC and CXX respectively.
 # cray-intel does not imply the native module in this case
 
+#module purge
```

Comment:
Why is this commented out?

Reply:
wcoss2 is not a regularly tested platform. 
Only a few people have access to it and it gets updates to it modules regularly.
I don't know what will break or what more will need to be loaded after a purge.
I was going to ask them to test with this and report back.
Easiest way to ask was to uncomment 1 line and run it.